stripe.php

```diff
@@ -275,3 +268,19 @@ function stripe_civicrm_managed(&$entities) {
 
   return _stripe_civix_civicrm_managed($entities);
 }
+
+/**
+ * Implementation of hook_civicrm_alterContent
+ *
+ * @return void
+ */
+function stripe_civicrm_alterContent( &$content, $context, $tplName, &$object ) {
+  if($context == 'form') {
+    $stripe_key = stripe_get_key($object);
+    if(empty($stripe_key)) {
+      return;
+    }
+    $stripeJSURL = CRM_Core_Resources::singleton()->getUrl('com.drastikbydesign.stripe', 'js/civicrm_stripe.js');
```

Comment:
hello @drastik ,
Thanks for checking out the patch.
So the reason why I removed addScriptFile() is because it simply doesn't work on webform. It works only for civicrm pages and hence stripe.js token error occurs on webforms.

The approach I used is to append civicrm_stripe.js at the end of content which means all the form fields are present already.
Yes I have seen this approach used on here: https://github.com/civicrm/civihr/blob/staging/hrui/hrui.php#L676
Many Thanks!